ui/src/components/Theme/index.tsx

```diff
@@ -0,0 +1,42 @@
+import {moonIcon, sunIcon} from '../../assets';
+import {useEffect, useState} from 'react';
+import styles from './styles.module.scss';
+
+const LIGHT_MODE_VALUE = 'light_mode';
+const DARK_MODE_VALUE = 'dark_mode';
+const WEBSITE_THEME = 'website_theme';
+
+export const Theme: any = () => {
```

Comment:
Try not to use `any` type unless necessary. You can use `React.FC` here.
Also, `Theme` doesn't sound like a component, let's make it something like `ThemeButton`.